res/values-ca/strings.xml

```diff
@@ -1042,7 +1042,7 @@
     <string name="Buy">Buy</string>
     <string name="Sell">Sell</string>
     <string name="community_skin_requirement">You must purchase Enable Custom Skins to use community skins.</string>
-    <string name="You_can_purchase_this_community_skin_for_">You can purchase this community skin for</string>
+    <string name="You_can_purchase_this_community_skin_for_">You can purchase this community skin for:</string>
```

Comment:
Here it isn't supposed to be double dot, but spacebar.

Reply:
Hi jei, sorry I didn't understand the second thing, what should I translate?